src/frontend/apps/impress/src/features/docs/doc-header/components/DocHeader.tsx

```diff
@@ -24,48 +22,43 @@ interface DocHeaderProps {
 }
 
 export const DocHeader = ({ doc, versionId }: DocHeaderProps) => {
-  const { colorsTokens } = useCunninghamTheme();
+  const { colorsTokens, spacingsTokens } = useCunninghamTheme();
+  const spacings = spacingsTokens();
+  const colors = colorsTokens();
+
   const { t } = useTranslation();
-  const { formatDate } = useDate();
+  const docIsPublic = doc.link_reach === LinkReach.PUBLIC;
+
   const { transRole } = useTrans();
-  const { isMobile, isSmallMobile } = useResponsiveStore();
 
   return (
     <>
-      <Card
+      <Box
         $width="100%"
-        $margin={isMobile ? 'tiny' : 'small'}
+        $padding={{ vertical: 'base' }}
+        $gap={spacings['base']}
         aria-label={t('It is the card information about the document.')}
       >
-        <Box
-          $padding={isMobile ? 'tiny' : 'small'}
-          $direction="row"
-          $align="center"
-        >
-          <StyledLink href="/">
-            <Text
-              $isMaterialIcon
-              $theme="primary"
-              $variation="600"
-              $size="2rem"
-              $css={css`
-                &:hover {
-                  background-color: ${colorsTokens()['primary-100']};
-                }
-              `}
-              $hasTransition
-              $radius="5px"
-              $padding="tiny"
-            >
-              home
-            </Text>
-          </StyledLink>
+        {docIsPublic && (
           <Box
-            $width="1px"
-            $height="70%"
-            $background={colorsTokens()['greyscale-100']}
-            $margin={{ horizontal: 'tiny' }}
-          />
+            aria-label={t('Public document')}
+            $color={colors['primary-600']}
+            $background={colors['primary-100']}
+            $radius={spacings['3xs']}
+            $direction="row"
+            $padding="xs"
+            $flex={1}
+            $align="center"
+            $gap={spacings['3xs']}
+            $css={css`
+              border: 1px solid var(--c--theme--colors--primary-300, #e3e3fd);
+            `}
+          >
+            <Icon data-testid="public-icon" iconName="public" />
+            <Text>{t('Public document')}</Text>
+          </Box>
+        )}
+        <Box $direction="row" $align="center" $width="100%">
```

Comment:
Should it be on the top ?
![image](https://github.com/user-attachments/assets/a7cf018f-0164-4758-ae7e-f016c6916ca4)